src/components/query-selector/export-type.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+import { Center, GridCol, Paper, rem, Select, Stack, Title } from '@mantine/core';
+import { useState } from 'react';
+import BuildQueryButton from './build-query-button';
+import { capitalize } from 'lodash';
+
+export type SupportedExportTypes = 'patient' | 'group' | 'system';
+
+export interface ExportTypeProps {
+  exportType: SupportedExportTypes;
+  dropdownData?: string[];
+}
+
+/*
+ * Component for selecting which type of export the user wants.
+ */
+export default function ExportType({ exportType, dropdownData }: ExportTypeProps) {
+  const [queryId, setQueryId] = useState<string | null>('');
+
+  return (
+    <GridCol span={{ base: 12, md: 6, lg: 3 }} style={{ minWidth: rem(600) }}>
+      <Paper shadow="xl" radius="xl" p="xl">
+        <Stack>
+          <Center>
+            <Title>{`${capitalize(exportType)}-level Export`}</Title>
+          </Center>
+          {exportType == 'system' ? (
```

Comment:
You can make this slightly more consider by using `??`! That allows you basically just to say "if" and have no "else". So you would do `{exportType == 'system' ?? (<Select ...`

Reply:
Oh also `===` instead of `==`